### ISSUE TITLE
python3-pipx-0.14.0.0_1: missing dependency python3-click

### ISSUE THREAD

**Issue by Mrnoname1000** [void-linux/void-packages#17563]
### System

* xuname:  
Void 5.4.2_1 x86_64 AuthenticAMD uptodate rrFFFFFF
* package:  
python3-pipx-0.14.0.0_1
python3-click-7.0_1

### Expected behavior
Pipx should have all dependencies installed.
### Actual behavior
Python3-click isn't installed; pipx throws an error when executed:
```
Traceback (most recent call last):
  File "/usr/bin/pipx", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3250, in <module>
    @_call_aside
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3234, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3263, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'click' distribution was not found and is required by userpath
```
### Steps to reproduce the behavior
Install python3-pipx, execute it. Behaviour is fixed after installing python3-click.

**Comment by NeelChotai** [comment]
Reproduced, pull request with fix opened: https://github.com/void-linux/void-packages/pull/17569

**Comment by NeelChotai** [comment]
Could you confirm if this has been fixed by cfba21b?

**Comment by xtraeme** [comment]
Already fixed
